src/AdminConsole/RateLimiting/AdminPageRateLimit.cs

```diff
@@ -0,0 +1,24 @@
+using System.Threading.RateLimiting;
+using Microsoft.AspNetCore.RateLimiting;
+using Passwordless.AdminConsole.Helpers;
+
+namespace Passwordless.AdminConsole.RateLimiting;
+
+public static class AdminPageRateLimit
+{
+    public const string PolicyName = "organizationIdFixedLength";
+    private const int PermitLimit = 100;
+    private static readonly TimeSpan Window = TimeSpan.FromMinutes(1);
+    private const int QueueLimit = 0;
+
+    public static RateLimiterOptions AddAdminPageRateLimitPolicy(this RateLimiterOptions limiter) =>
+        limiter.AddPolicy(PolicyName, context =>
+            RateLimitPartition.GetFixedWindowLimiter(
+                context.User.GetOrgId()!.Value.ToString(),
```

Comment:
When can `.GetOrgId()` return null?

Reply:
If the user does not have the "OrgId" claim, which would occur if the user is unauthenticated (in which they would be redirected to the login page) or if they were never added to an org (which is very highly unlikely). 

TLDR: it shouldn't happen.